tests/org.eclipse.n4js.lang.tests/src/org/eclipse/n4js/tests/parser/ES_07_08_3_NumericLiteralEsprimaTest.xtend

```diff
@@ -187,7 +187,7 @@ class ES_07_08_3_NumericLiteralEsprimaTest extends AbstractParserTest {
 		val script = '0O127;'.parseSuccessfully
 		val statement = script.scriptElements.head as ExpressionStatement
 		val number = statement.expression as OctalIntLiteral
-		assertEquals(0127, number.toInt)
+		assertEquals(87, number.toInt)
```

Comment:
I recommend to keep the octal literal in the expectation

Reply:
@szarnekow Thanks for your review! Keeping the octal literal cause the assert to fail!

Reply:
Interesting. I found the change in the Xbase codebase, that caused octal literals to work and stop working. There is such thing as a side-effect-free bugfix :)

Reply:
so, are we keeping `87` or are we waiting for xbase update and then we go with the original test?

Reply:
Xbase never had octal number literals, it only was a bug in the compiler

From the spec:
```
Number Literals
Xbase supports roughly the same number literals as Java with a few notable differences. As in Java 7, you can separate digits using _ for better readability of large numbers. An integer literal represents an int, a long (suffix L) or even a BigInteger (suffix BI). There are no octal number literals.
```

Reply:
So there won't be a fix. We either have to keep the 87, or use `Integer.parseInt(127, 8)` which I'd prefer

Reply:
done